surveys/2021-annual-survey/questions.md

```diff
@@ -288,3 +407,103 @@ Is Rust mandated for your course or activity, or did you choose it yourself?
 - Optional (but suggested in some way)
 - Optional (completely driven by you)
 
+
+## Rust community
+
+### What resources do you use to keep up with the Rust ecosystem?
+
+Select all that apply:
+
+> **REMOVED**
+>
+> Integrated with 'How do you participate in the community' question
+
+
+## Rust conferences
+
+### Did you attend a Rust conference in the past year? If so, which ones?
+
+Free form
+
+> **REMOVED**
+>
+> Vague, not useful. Somewhat replaced by community participation question.
+
+### If you wanted to attend a Rust conference but couldn't, why not?
+
+Free form
+
+> **REMOVED**
+>
+> Vague, not useful.
+
+### If you are interested in attending a Rust conference, which of these regions would you travel to?
+
+Select all that apply
+
+> **REMOVED**
+>
+> Not useful.
```

Comment:
Are we sure this is not useful? For regional conferences, this might give them an indication of how likely their conference is to draw a crowd from outside their region. For non-regional conferences (e.g., RustFest) this can give them an indicator of additional places they may want to look into hosting. 

Overall, I can see why you'd think this isn't useful, but perhaps we should ship it around to conference organizers to get their input.

Reply:
From previous conversations, most conference organisers start with a location, usually local, rather than thinking "I want to put in a conference, where should I do it". I think this would only be useful if the foundation or maybe rustfest wanted to do another conference